static/js/components/SpinnerButton.js

```diff
@@ -24,7 +24,7 @@ export default class SpinnerButton extends React.Component {
     }
   }
 
-  componentWillReceiveProps(nextProps: SpinnerButtonProps) {
+  getDerivedStateFromProps(nextProps: SpinnerButtonProps) {
```

Comment:
This is causing a console warning. Can you fix?

![Screenshot from 2019-09-10 17-22-11](https://user-images.githubusercontent.com/863262/64651476-96134300-d3ef-11e9-86a1-0e88f13ed4c4.png)